koku/api/report/aws/aws_query_handler.py

```diff
@@ -92,6 +92,38 @@ def _format_query_response(self):
 
         return output
 
+    def _build_sum(self, query, query_sum):
```

Comment:
query_sum probably shouldn't be a parameter since it is created entirely in the method and is the return value.

Reply:
Its actually created here:
https://github.com/project-koku/koku/pull/705/files#diff-a6f074cb1e0e6223367a07a8b3dbe9e3R134

But I'll just move this init into this function since that is the only alteration.

Reply:
updated.

Reply:
Ah, sorry about that.